docs/Conditions.md

```diff
@@ -0,0 +1,46 @@
+# Conditions
+
+> This document is incomplete
+
+- [If](#if)
```

Comment:
I forgot this statement completely,  
Can you also add this to the TODO in the readme.

Reply:
Added :-)